model/visor/processing.go

```diff
@@ -184,6 +191,30 @@ func (pl ProcessingActorList) PersistWithTx(ctx context.Context, tx *pg.Tx) erro
 	return nil
 }
 
+func (pl ProcessingActorList) UpdateWithTx(ctx context.Context, tx *pg.Tx) error {
```

Comment:
There is a test in the storage package `TestMarkActorComplete`. Maybe there is a way to keep that working against this code.